File: Daily Test/Python/Pattern - Lower Left Plus Symbol 13.4.2026.py
m = int(input())
num = m

for i in range(1, m+1):
    
    # print '+'
    for j in range(1, i):
        print("+", end=" ")
    
    # print numbers
    for j in range(num):
        print(num, end=" ")
    
    print()
    num -= 1

File: Daily Test/C++/Pattern - Lower Left Plus Symbol 13.4.2026.cpp
#include <iostream>
using namespace std;

int main() {
    int m;
    cin >> m;

    int num = m;

    for(int i = 1; i <= m; i++) {

        for(int j = 1; j < i; j++) {
            cout << "+ ";
        }

        for(int j = 1; j <= num; j++) {
            cout << num << " ";
        }

        cout << endl;
        num--;
    }
}